demos/dotnet/dotnet/dotnet.cs

```diff
@@ -157,8 +180,24 @@ public static ProjectProperties InitializeProperties(string[] args, Log log, boo
         properties.OutputType = buildDll ? ".dll" : ".exe";
         FindCompiler(properties);
 
+        var projectFiles = Directory.GetFiles(properties.ProjectDirectory, "*.dotnetproj");
+        bool projectFileExists = projectFiles.Length > 0;
         // Sources
-        properties.Sources.AddRange(Directory.GetFiles(properties.ProjectDirectory, "*.cs"));
+        if (projectFileExists)
+        {
+            foreach(var projectFile in projectFiles)
+            {
+                properties.Sources.AddRange(ParseProjectFile(properties.ProjectDirectory, projectFile));
```

Comment:
I would rather pass properties to the parse method. We will have more things in the project file besides sources (very soon).